### DISCUSSION TITLE
Unable to get Postgres app/source working

### DISCUSSION THREAD

**Discussion by spicypenguin** [grouparoo/grouparoo#1350] [Chat]
I have followed the instructions to try and setup Node.js locally and get Grouparoo running.

Everything appears to be working correctly, grouparoo is up and I've configured an app (local postgres), a source (users table within postgres), and properties (one for each field on the table).

However I'm not getting any profiles imported and hitting a lot of errors in the /resque tab, but the error messages are not particularly helpful.

```
on Feb 22 2021 20:49:59 GMT-0800 (Pacific Standard Time) | ➕ TypeError: Cannot read property 'methods' of null | profileProperties | profileProperties:enqueue | jefftop-pc:3795+1 | {}
-- | -- | -- | -- | -- | --
```
Any ideas what is causing this error?

**Comment by bleonard** [comment]
Likely cause [here](https://github.com/grouparoo/grouparoo/blob/b08d29a7490eeedd2c34e9c30ac51d0bdcda89d5/core/src/tasks/profileProperty/enqueue.ts#L36):
`const { pluginConnection } = await this.getPlugin();` is returning null.

My best guess is that you need to run `grouparoo install @grouparoo/postgres` 
In the past, this was in the example, but now it need to be done explicitly. I'll update the documentation.

Thanks for your help.
Let me know how it goes.

**Reply by bleonard** [reply]
Here's the doc on install: https://www.grouparoo.com/docs/installation/plugins

I'd love feedback on the documentation. Did you got through, the [steps](https://www.grouparoo.com/docs/config/apps/community) to add an app?
Commands like this: `grouparoo generate postgres:app data_warehouse`

That failed for me (as expected) without `grouparoo install @grouparoo/postgres`, so maybe that's not what you did.

The above error would happen if you edited the files directly. Is that what happened?

**Reply by evantahler** [reply]
@spicypenguin if you are getting those errors in `/resque`, could you please expand the error (there's a `+` button) so you can share the whole stack trace?  Thanks!

**Comment by evantahler** [comment]
If this problem was related to not having the `@grouparoo/postgres` plugin installed,  https://github.com/grouparoo/grouparoo/pull/1371 will help with a better error message.

**Comment by evantahler** [comment]
Ping! Just checking in @spicypenguin - anything we can do to help?
